VK_App/VK_App/ViewControllers/LoginView.swift

```diff
@@ -0,0 +1,114 @@
+// LoginView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+import WebKit
+
+/// Вью входа в приложение
+final class LoginView: UIView {
+    // MARK: IBOutlet
+
+    @IBOutlet var scrollView: UIScrollView!
+    @IBOutlet var signInWithAppleButton: UIButton!
+    @IBOutlet var loginTextField: UITextField!
+
+    // MARK: Public Visual Components
+
+    var firstDoteView = UIView()
+    var secondDoteView = UIView()
+    var thirdDoteView = UIView()
+
+    // MARK: Public Methods
+
+    @objc func keyboardWillShownAction(notification: Notification) {
+        guard let info = notification.userInfo as? NSDictionary else { return }
+        guard let kbSize = (info.value(forKey: UIResponder.keyboardFrameEndUserInfoKey) as? NSValue)?.cgRectValue.size
+        else { return }
+
+        let contentInsets = UIEdgeInsets(top: 0.0, left: 0.0, bottom: kbSize.height, right: 0)
+        scrollView.contentInset = contentInsets
+        scrollView.scrollIndicatorInsets = contentInsets
+
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(hideKeyboardAction))
+        scrollView.addGestureRecognizer(tapGesture)
+    }
+
+    @objc func hideKeyboardAction() {
+        scrollView.endEditing(true)
+    }
+
+    @objc func keyboardWillHideAction(notification: Notification) {
+        scrollView.contentInset = UIEdgeInsets.zero
+        scrollView.scrollIndicatorInsets = UIEdgeInsets.zero
+    }
+
+    func startAnimationView() {
+        firstDoteView = setupSubView(newView: firstDoteView, xPosition: -20, yPosition: -115)
+        secondDoteView = setupSubView(newView: secondDoteView, xPosition: 0, yPosition: -115)
+        thirdDoteView = setupSubView(newView: thirdDoteView, xPosition: 20, yPosition: -115)
```

Comment:
```suggestion
        firstDoteView = setupSubView(newView: firstDoteView, xPosition: -, yPosition: -)
        secondDoteView = setupSubView(newView: secondDoteView, xPosition: 0yPosition: -)
        thirdDoteView = setupSubView(newView: thirdDoteView, xPosition:, yPosition: -)
```
убираем магические числа